src/plugins/condition/criterion/TelemetryCriterion.js

```diff
@@ -77,7 +77,7 @@ export default class TelemetryCriterion extends EventEmitter {
         let result = false;
         params.push(data[this.metadata]);
         if (this.input instanceof Array && this.input.length) {
```

Comment:
What is the use case where this.input is something different than an array? You might want to consider this.input to always be an array, especially since it is instantiated as [].

This is more my preference, but for arrays I like to pluralize because it gives more clarity as to what the var is.

Reply:
I can think of no use case when it's not an array. If it's 0 or 1 it's still an array. I changed it to just check the length. Changing the name to inputs seems misleading since there is usually only one in the array.